index.js

```diff
@@ -153,9 +155,25 @@ module.exports = function (app) {
                   ]
                 }
 
-                // app.debug("got delta: " + JSON.stringify(delta))
+                app.debug('got delta: ' + JSON.stringify(delta))
```

Comment:
Please don't do dubug like this. JSON.stringify will get called even if debug is off.

Do ` app.debug('got delta: %j', delta)`